.github/workflows/push-on-develop.yml

```
@@ -35,19 +44,71 @@ jobs:
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+
+      - name: Create and push manifest list 📦
+        run: |
+          docker buildx imagetools create --tag ${{ env.REGISTRY_IMAGE }}:develop \
+            $(for digest in /tmp/digests/*-digest.txt; do cat "$digest"; done | tr '\n' ' ')
+
+      - name: Inspect manifest list 🕵️
+        run: |
+          docker buildx imagetools inspect ${{ env.REGISTRY_IMAGE }}:develop
+
+    deploy-to-preproduction:
```

actionlint

unexpected key "deploy-to-preproduction" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)